kfp/doc/setup.md

```diff
@@ -74,7 +74,7 @@ You can create a Kind cluster with all required software installed using the fol
 from this main package directory or from the `kind` directory.
 If you do not want to upload the testing data into the locally deployed Minio, and reduce memory footprint, please set:
 ```bash
-export POPULATE_TEST_DATA ?= 0
+export POPULATE_TEST_DATA=0
```

Comment:
I approved, but I really do not like = without ?

Reply:
it is in the command line, and not in the Makefile